### PR TITLE
Feature/deeplinking

### PR DESCRIPTION
**Pull request by totorototo**
work in progress. do not merge right now! 
Authenticate saga not finished yet. @guijesuis

### CODE REVIEW

File: app/deeplink.js
Comment (guijesuis) [inline app/deeplink.js]: no!!!

Comment (guijesuis) [reply app/deeplink.js]: we should parse query -> no specific here

**Review by guijesuis — CHANGES_REQUESTED** [review]

File: app/deeplink.js
Comment (totorototo) [reply app/deeplink.js]: got your point!

**Comment by totorototo** [comment]
changes done! to be merged after being reviewed.

**Comment by totorototo** [comment]
waiting for your approval, Rick!

**Pull request merged**
